src/Symfony/Component/Validator/ValidatorBuilder.php

```diff
@@ -28,16 +27,22 @@
 use Symfony\Component\Validator\Mapping\Loader\StaticMethodLoader;
 use Symfony\Component\Validator\Mapping\Loader\XmlFileLoader;
 use Symfony\Component\Validator\Mapping\Loader\YamlFileLoader;
+use Symfony\Component\Validator\Util\LegacyTranslatorProxy;
 use Symfony\Component\Validator\Validator\RecursiveValidator;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\TranslatorTrait;
 
 /**
  * The default implementation of {@link ValidatorBuilderInterface}.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @final since Symfony 4.2
```

Comment:
The since part can be removed.

Reply:
Actually it is required: we should add "since..." to tell that this is not yet really final and that BC should still be preserved until 5.0. Using only `@final` means we can change the interface *now*, which would be wrong here.

Reply:
indeed, I read `@since` instead of `@final`